pkg/controller/binding/binding_controller.go

```diff
@@ -525,6 +523,14 @@ func (r *ReconcileBinding) deleteSecret(secret *corev1.Secret) error {
 	return nil
 }
 
+func getSecretName(instance *ibmcloudv1alpha1.Binding) string {
+	secretName := instance.ObjectMeta.Name
+	if instance.Spec.SecretName != "" {
+		secretName = instance.Spec.SecretName
+	}
+	return secretName
+}
+
```

Comment:
This looks good. The default should be the same name as the binding name, but if `secretName` is defined it should use that.